tests/test_item_resource.py

```diff
@@ -0,0 +1,196 @@
+        self.item_2 = Item('Item two', 'this is item two',
+                           self.bucketlist_with_items_id)
+        db.session.add(self.item_1)
+        db.session.add(self.item_2)
+        db.session.commit()
+
+        self.item_one_id = Item.query.filter_by(title='Item one').first().id
+
+        self.response = self.client.post(
+            '/api/v1/auth/login', data=json.dumps(self.user),
+            headers=self.headers)
+        self.response_content = json.loads(self.response.data)
+        self.headers['Authorization'] = 'JWT {}'.format(
+            self.response_content['access_token'])
+
+    def tearDown(self):
+        super(ItemResourceTest, self).tearDown()
+
+    #view items
+    def test_item_names_returned_by_view_bucketlists(self):
+        response = self.client.get(
+            '/api/v1/bucketlists/{}/items'.format(
+                self.bucketlist_with_items_id),
+            headers=self.headers)
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue(b'Item one' in response.data)
+        self.assertTrue(b'Item two' in response.data)
+
+    def test_cannot_view_items_without_token(self):
+        no_token = self.headers
+        no_token['Authorization'] = ""
+        response = self.client.get(
+            '/api/v1/bucketlists/{}/items'.format(
+                self.bucketlist_with_items_id),
+            headers=no_token)
+        self.assertTrue(b'Authorization Required' in response.data)
+
+    def test_response_message_for_non_existent_id(self):
+        response = self.client.get(
+            '/api/v1/bucketlists/2000/items', data={},
+            headers=self.headers)
+        print(response.data)
+        self.assertTrue(b'bucketlist does not exist' in response.data)
+
+    #view item
+    def test_item_name_returned_when_item_id_is_specified(self):
+        response = self.client.get(
+            '/api/v1/bucketlists/{}/items/{}'.format(
+                self.bucketlist_with_items_id, self.item_one_id),
+            headers=self.headers)
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue(b'Item one' in response.data)
+
+    def test_message_if_item_id_does_not_exist(self):
+        response = self.client.get(
+            '/api/v1/bucketlists/{}/items/{}'.format(
+                self.bucketlist_with_items_id, 555),
+            headers=self.headers)
+        self.assertTrue(b'item does not exist' in response.data)
+
+    #test create item
+    def test_item_created_successfully_message(self):
+        new_item = {
+            "title": "title one",
+            "description": "this is my first title"
+        }
+        response = self.client.post(
+            '/api/v1/bucketlists/{}/items'.format(self.item_one_id),
+            data=json.dumps(new_item),
+            headers=self.headers)
+        print(response.data)
+        self.assertTrue(b'item created successfully' in response.data)
```

Comment:
When you test for `POST` operations, it's also advisable to query the database and assert that the item has been created there as well.